tests/endToEnd.py

```diff
@@ -212,13 +218,15 @@ def testEndToEnd(self):
         #        datarel needs imsim_*.
         ver = 'imsim-2010-12-17-1'
         print "Setting up astrometry_net_data", ver
+        # XXX what is actually used from this setup -- a path in the env?
```

Comment:
Yes, but is that meant to correspond to a directory inside the astrometry_net_data repository? Or is that a version string that eups understands maps to a data set in some other location?